tools/alpine/detect-arch-var

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env sh
```

Comment:
Great! I will set the DOCKER_CONTENT_TRUST=0 as default in my PR, it will be easy to be tuned once this issue fixed.